region/crateria/central.json

```
@@ -111,6 +115,7 @@
           "name": "Ship",
           "nodeType": "door",
           "nodeSubType": "elevator",
+          "nodeAddress": "",
```

Was that forgotten or missing? Do we want a FIXME note?

---

Oh, it's the ship. Does it need an address?

---

```suggestion
          "note":  "FIXME If there's a proper memory address to the ship/elevator it was in hiding",
```

(I would remove the nodeAddress line completely for now)